kw is not defined

Hey @NathanaelA,

@jlooper and I were giving this generator a shot, and we got this error at the very end of the process. Any idea what’s up?

```
Generating readme.me...
Generating package.json...
readline.js:925
            throw err;
            ^

ReferenceError: kw is not defined
    at generatePackage (/Users/tj/Desktop/MyNativeScriptPlugin/setup.js:582:16)
    at /Users/tj/Desktop/MyNativeScriptPlugin/setup.js:220:9
    at /Users/tj/Desktop/MyNativeScriptPlugin/setup.js:304:13
    at /Users/tj/Desktop/MyNativeScriptPlugin/setup.js:360:17
    at Interface._onLine (readline.js:209:5)
    at Interface._line (readline.js:550:8)
    at Interface._ttyWrite (readline.js:827:14)
    at ReadStream.onkeypress (readline.js:106:10)
    at emitTwo (events.js:87:13)
    at ReadStream.emit (events.js:172:7)

npm ERR! Darwin 15.6.0
npm ERR! argv "/usr/local/Cellar/node4-lts/4.4.4/bin/node" "/usr/local/bin/npm" "start"
npm ERR! node v4.4.4
npm ERR! npm  v2.15.1
npm ERR! code ELIFECYCLE
npm ERR! tns-template-plugin@1.0.6 start: `node setup.js`
npm ERR! Exit status 1
```

--- a/setup.js
+++ b/setup.js
@@ -579,7 +579,7 @@ function generatePackage(answers) {
     if (answers.keywords.length) {
         var keywords = answers.keywords.split(",");
         for (var i=0;i<keywords.length;i++) {
-            kw = keywords[i].trim();
+            var kw = keywords[i].trim();
             if (kw.length > 0) {
                 data.keywords.push(kw);
             }

